src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea() + " sq. units,"
+                + " firstLeg: " + firstLeg + " units," + " secondLeg: " + secondLeg + " units,"
+                + " color: " + getColor());
```

Comment:
The message in the `draw` method is incorrect. It mentions 'isosceles trapezoid', but this is a `RightTriangle` class. Please correct the figure type in the output message.